docker-compose.openSearch.yml

```diff
@@ -28,15 +28,40 @@ services:
       - "OPENSEARCH_HOSTS=${OPENSEARCH_HOSTS:-http://opensearch:9200}"
       - "ELASTICSEARCH_USERNAME=${ELASTICSEARCH_USERNAME:-admin}"
       - "ELASTICSEARCH_PASSWORD=${ELASTICSEARCH_PASSWORD:-admin}"
+      - 'SERVER_BASEPATH="/opensearch"'
+      - 'SERVER_REWRITEBASEPATH=true'
     volumes:
-      - "./config/kibana.yml:/usr/share/opensearch-dashboards/config/kibana.yml"
+      - "./opensearch.yml:/usr/share/opensearch-dashboards/config/opensearch.yml"
     networks:
       openimis-net:
         ipv4_address: 172.20.${IP_SUB:-20}.97
 
+  nginx:
```

Comment:
Is this general use nginx compatible what what is used currently as a gateway? On test server "gateway" service is combined with the frontend container. I'd like to know if this could cause any compatibility issues.

Reply:
There shouldn't be any compatibility issues. I checked locally having full dockerized instance with runned db, backend, frontend and opensearch components services including additional nginx.

Reply:
No, I don't this such combination can destroy dockerized solution. 

```
docker compose -f docker-compose.yml -f docker-compose.openSearch.yml build db backend frontend opensearch opensearch-dashboards nginx

docker compose -f docker-compose.yml -f docker-compose.openSearch.yml up db backend frontend opensearch opensearch-dashboards nginx
```

such combinations works fine in the current docker-compose structure (Today I updated this) and there were no problems while running such services.
![Peek 2023-07-04 14-06](https://github.com/openimis/openimis-dist_dkr/assets/52816247/4795d88a-47fa-4935-8c2c-6284e0b77cef)
Moreover as you can see - it works in that configuration. Also you can take a look into current docker-compose - gateway is used within frontend as you mention and there is no more such service.